tests/tests/test_backup_alerts.py

```diff
@@ -2,11 +2,11 @@
 import random
 import time
 import datetime
-import alerts
-import settings
-import kubectl
-import clickhouse
-import util
+import tests.alerts as alerts
```

Comment:
why this rename was necessary?

Reply:
It was necessary so that I don't refactor all in-code mentions

Reply:
i mean we shall rename `tests/tests` to something like `tests/e2e`

and change regression.py to load('e2e.test')